Server/src/services/track.service.ts

```diff
@@ -124,7 +127,11 @@ export default class TrackService {
 
 		// for easier access we set the property of track kilometer to the already calculated value
 		if (projectedPoint.properties["location"] == null) {
-			// TODO: log this
+			logger.error(
+				`Turf did something wrong while computing nearest point on line for track with id ${
```

Comment:
Maybe rather something like: Turf error: Could not compute nearest point on line for track with id ... and position ...